hipcheck/src/metric/mod.rs

```diff
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
-pub mod affiliation;
-pub mod binary;
+
+ 
```

Comment:
A lot of the lines you've deleted are replaced with empty lines, but I'd like to have them removed entirely and not have odd empty spaces in the codebase. You might look into `rustfmt` to help format your code before committing.